power-platform/admin/managed-environment-sharing-limits.md

```diff
@@ -39,6 +39,44 @@ If a user tries to share a canvas app that contradicts the sharing rules, they w
 
 :::image type="content" source="media/managed-environment-canvas-app-sharing-rule.png" alt-text="Screenshot of a message when canvas app doesn't respect sharing rule.":::
 
+## Use PowerShell to set sharing limits
+
+You can also use PowerShell to set and remove sharing limits.
+
+### Set sharing limits
+
+Here's an example PowerShell script that excludes sharing with security groups and limits to 20 the number of individuals canvas apps can be shared to, for an existing managed environment:
```

Comment:
"Here's a PowerShell script that excludes sharing a canvas app with security groups and excludes sharing a canvas app with more than 20 individuals within a managed environment:"